### PR TITLE
Add nyc coverage reporting with `npm run coverage`

### PR DESCRIPTION
**Pull request by foolip**
This is mostly to easily see which files are entirely untested.

### CODE REVIEW

**Comment by foolip** [comment]
Found some dead code: https://github.com/web-platform-tests/wpt-pr-bot/pull/114

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules/
 .DS_Store
+.nyc_output/
```
Comment (jugglinmike) [inline .gitignore]: The current invocation doesn't generate any files

Comment (foolip) [reply .gitignore]: Oh, it might have been created by one of my failed attempts before I got it right.

**Comment by foolip** [comment]
Running this in CI would be trivial, but I don’t know that it’ll really be useful on an ongoing basis. Since there are a lot of transient dependencies which might get `npm audit` issues, maybe it’s not worth it.

This was trivial to set up, so I’ll close and reopen at a later time if I see the need.

The future of this project won’t really be clear until we try to migrate some of the key pieces and see what we run into beyond read-only tokens for pull requests.

**Comment by foolip** [comment]
Now that https://github.com/web-platform-tests/wpt-pr-bot/pull/116 has landed it'll be easier (for me) to put this into CI, so reopening.

**Comment by foolip** [comment]
Oh, can't reopen because "The nyc-coverage branch was force-pushed or recreated." I'll open a new PR.

File: .gitignore (hunk 1 above)
Comment (foolip) [reply .gitignore]: Nope, in creating a new PR I still see this directory being created, so I'll leave this.

**Comment by foolip** [comment]
Sent https://github.com/web-platform-tests/wpt-pr-bot/pull/118

**Pull request closed without merging**